js/Makefile

```diff
@@ -16,6 +16,7 @@ APP_CONFIG ?= development
 IOS_APP_TARGET ?= Berty Debug
 ANDROID_APP_TARGET ?= Debug
 IOS_DEVICE ?= iPhone 11
+REQUIRED_JAVA_VER ?= 18
```

Comment:
shouldn't it be hardcoded instead of configurable?

Reply:
Not sure about this, I can hardcode it first and we'll see later?

Reply:
I'm more interested to have an example of use case where it's a good idea to make this overridable?